packages/components/board/stories/board.stories.js

```diff
@@ -5,14 +5,19 @@ import '@tradeshift/elements.board';
 
 storiesOf('ts-board', module)
 	.addDecorator(withKnobs)
-	.add('default', () => {
-		const dir = text('ltr', 'Dir');
-		const title = text('My Board', 'Title');
-		const content = text('Put any html inside the board', 'Content');
-
-		return html`
-			<ts-board dir="${dir}" data-title="${title}">
-				${content}
-			</ts-board>
-		`;
-	});
+	.add(
+		'default',
+		() => {
+			const dir = text('ltr', 'Dir');
+			const title = text('My Board', 'Title');
+			const mainDiv = document.createElement('div');
+			mainDiv.slot = 'main';
+			mainDiv.innerHTML = text('Put any html inside the board', '<h3>Content</h3>');
```

Comment:
Same here we should have a consistent way to demonstrate slots in stories. I don't have any specific idea right now, just another thing to discuss in tech talk

Reply:
Yeah, we definitely should make the `slot` feature clear to developers.
Actually, in board we have only one slot, so it could be a default one, without names, so everything that placed inside the `ts-board` will be placed in this slot. But it would be breaking changes.

Reply:
Yes, one concern that we had before was for developers who are not familiar with the concept of slots, if we make changes and add another slot for example, it would be confusing for them to have the default slot and next to that named slot, probably that's the reason Leo used named slot. But I think we should use default slot and have a clear documentation with examples in those cases.